demisto_sdk/commands/lint/docker_helper.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import os
+import tarfile
+import tempfile
+from pathlib import Path
+from typing import Dict, List, Optional, Tuple, Union
+
+import docker
+from docker.types import Mount
+
+DOCKER_CLIENT = None
+logger = logging.getLogger('demisto-sdk')
+PATH_OR_STR = Union[Path, str]
+
+
+def init_global_docker_client(timeout: int = 60, log_prompt: str = ''):
+
+    global DOCKER_CLIENT
+    if DOCKER_CLIENT is None:
+        try:
+            logger.info(f'{log_prompt} - init and login the docker client')
+            DOCKER_CLIENT = docker.from_env(timeout=timeout)
+            docker_user = os.getenv('DOCKERHUB_USER')
+            docker_pass = os.getenv('DOCKERHUB_PASSWORD')
+            DOCKER_CLIENT.login(username=docker_user,
+                                password=docker_pass,
+                                registry="https://index.docker.io/v1")
+        except Exception:
+            logger.exception(f'{log_prompt} - failed to login to docker registry')
+
+    return DOCKER_CLIENT
+
+
+def copy_file(cp_from: PATH_OR_STR, cp_to: PATH_OR_STR) -> Path:
+    cp_from = Path(cp_from)
+    cp_to = Path(cp_to)
+    cp_to.touch()
+    cp_to.write_bytes(cp_from.read_bytes() if cp_from.exists() else b'')
+    return cp_to
+
+
+class Docker:
+
+    @staticmethod
+    def pull_image(image: str) -> docker.models.images.Image:
+        docker_client = init_global_docker_client()
+        try:
+            return docker_client.images.get(image)
+        except docker.errors.ImageNotFound:
+            return docker_client.images.pull(image)
+
+    @staticmethod
+    def get_mounts(files: List[Tuple[str, PATH_OR_STR]]) -> List[Mount]:
+        """
+        Args:
+            files: a list of (target path in container, source path in machine).
+        Returns:
+            a list of mounts
+        """
+        mounts = []
+        for target, src in files:
+            try:
+                src = Path(src)
+                if src.exists():
+                    mounts.append(Mount(target, str(src.absolute()), 'bind'))
+            except Exception:
+                logger.debug(f'Failed to mount {src} to {target}')
+        return mounts
+
+    @staticmethod
+    def copy_files(container: docker.models.containers.Container, files: List[Tuple[str, PATH_OR_STR]]):
+        """
+        Args:
+            container: the container object.
+            files: a list of (target path in container, source path in machine).
+        Returns:
+            a list of mounts
```

Comment:
please explain what this method does (its name suggests it copies files. from where to where? and if so, what `mounts` are returned?)